ci/vm.sh

```diff
@@ -59,12 +59,12 @@ make_disk() {
 }
 
 gen_metadata() {
-	local cprcmd hardware_id id
-	id=$(uuidgen)
+	local cprcmd hardware_id
 	hardware_id=$(uuidgen)
```

Comment:
I feel like handling the input variables should come first. 

hardware_id should be made a local, and moved closer to it's first usage.